docker/local/entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e  # Останавливаем скрипт при ошибке
+
+# Устанавливаем зависимости
+composer install
+
+# Устанавливаем Roadrunner
+if [ ! -f ./rr ]; then
+    ./vendor/bin/dload get
+fi
+
+if [ ! -f ./.rr.yaml ]; then
+    cp .rr.yaml.example .rr.yaml
+fi
+
+if [ ! -f ./.env ]; then
+    cp .env.example .env
+fi
+
+
+# Генерируем новый ключ приложения
+php artisan key:generate
```

Comment:
The key will be regenerated on every run. Composer deps will also be installed every time. Is this really what we expect?

For example, sometimes I can run commands like `docker compose run ... `

Reply:
I mean if we install this template using `composer create-project` it would fire `post-root-package-install` + `post-create-project-cmd ` hooks. And Key would be generated it self.

https://getcomposer.org/doc/articles/scripts.md